test/unit/spec/LoginRouter_spec.js

```diff
@@ -2244,4 +2246,22 @@ Expect.describe('LoginRouter', function() {
       });
     });
   });
+
+  itp('should not be visible until initial render', async function() {
```

Comment:
add a test for non happy path..

Reply:
Looks like there is no proper error handling for v1 in case of error in `fetchInitialData ` - controller will not be rendered
https://github.com/okta/okta-signin-widget/blob/551c766ae892bb62f5b5c2ee32b2908fb9cc7128/src/util/BaseLoginRouter.js#L275-L277

Reply:
I have created a new issue OKTA-432776 for adding the error handling and related tests